node/test-helpers/subsystem/src/lib.rs

```diff
@@ -170,8 +170,11 @@ impl<M: Send + 'static, S: Spawn + Send + 'static> SubsystemContext for TestSubs
 		self.rx.next().await.ok_or(SubsystemError)
 	}
 
-	async fn spawn(&mut self, s: Pin<Box<dyn Future<Output = ()> + Send>>) -> SubsystemResult<()> {
-		self.spawn.spawn(s).map_err(Into::into)
+	async fn spawn(&mut self, name: &'static str, s: Pin<Box<dyn Future<Output = ()> + Send>>)
+		-> SubsystemResult<()>
+	{
+		self.spawn.spawn(name, s);
+		Ok(())
```

Comment:
looks like the result or `spawn` is ignored here compared to the previous revision or am I missing something?
i.e. should it have a question mark after spawn?
this is test code, maybe it doesn't matter, just curious